actors/builtin/multisig/multisig_test.go

```diff
@@ -165,6 +167,154 @@ func TestPropose(t *testing.T) {
 		})
 
 		actor.assertTransactions(rt)
+		rt.Verify()
+	})
+}
+
+func TestApprove(t *testing.T) {
+	actor := msActorHarness{multisig.MultiSigActor{}, t}
+
+	receiver := newIDAddr(t, 100)
+	anne := newIDAddr(t, 101)
+	bob := newIDAddr(t, 102)
+	chuck := newIDAddr(t, 103)
+
+	const noUnlockDuration = int64(0)
+	const numApprovals = int64(2)
+	const txnID = int64(0)
+	var sendValue = abi.NewTokenAmount(10)
+	var nilParams = abi.MethodParams{}
+	var signers = []addr.Address{anne, bob}
```

Comment:
FWIW I think these could be merged with the prior block as a `TestTransactions`.